File: All Codes/URI_1051.c
#include <stdio.h>
int main()
{
    double b,c;
    scanf("%lf",&b);
    if (b>=0 && b<=2000)
    {
        printf("Isento\n");
    }
    else if(b>=2000.01 && b<=3000)
    {
        c=(b-2000)*.08;
        printf("R$ %.2lf\n",c);
    }
    else if (b>3000 && b<4500)
    {
        c=((b-3000)*.18)+80;
        printf("R$ %.2lf\n",c);
    }
    else
    {
        c=((b-4500)*.28)+350;
        printf("R$ %.2lf\n",c);
    }
    return 0;
}

File: All Codes/URI_1051.py
a = float(input())

if a <= 2000.00:
    print("Isento")
elif a <= 3000.00:
    print(f"R$ {(a - 2000.00) * 0.08:.2f}")
elif a <= 4500.00:
    print(f"R$ {(a - 3000.00) * 0.18 + 80:.2f}")
else:
    print(f"R$ {(a - 4500.00) * 0.28 + 350:.2f}")